.gitignore

```diff
@@ -1 +1,2 @@
 venv/
+*.egg-info/
```

Comment:
Please put one and only one `\n`at the end of the file so the GitHub editor removes that ⛔